src/handlers/Handler.ts

```diff
@@ -56,6 +59,8 @@ export class Handler implements IHandler {
 	}
 
 	public async CreateReply(): Promise<void> {
+		const cliName = this.args?.[1] || '';
```

Comment:
could we give them better naems like initial name or something ? or can we just make one object that have both values